### ISSUE TITLE
module object has no attribute get_logger

### ISSUE THREAD

**Issue by Te-k** [decalage2/oletools#101]
Hi,

When using the last version of olevba, I have this error : 
```
Traceback (most recent call last):
  File "olevba.py", line 264, in <module>
    import ppt_parser
  File "/home/user/tools/docs/oletools/oletools/ppt_parser.py", line 54, in <module>
    log = olefile.get_logger('ppt')
AttributeError: 'module' object has no attribute 'get_logger'
```

In ppt_parser.py l54, I replaced 
```py
log = olefile.get_logger('ppt')
```
with
```py
log = logging.getLogger(__name__)
```
Which is not the best solution but works temporarily for me

**Comment by decalage2** [comment]
Hi @Te-k, which version of Python are you using, and on which OS?

And also, which version of oletools is it? Thank you.

**Comment by Te-k** [comment]
Python 2.7.12 on Ubuntu, oletools from the repository (commit 7b213e494204cd562e6676b250c459e8946eabc4)

**Comment by obilodeau** [comment]
I can confirm this also.

* Arch Linux
* Python 2.7.12
* oletools 0.50 installed from blackarch repository

**Comment by christian-intra2net** [comment]
This is probably caused by that strange import construct of olefile in ppt_parser. Might be fixed with with pull request #111. May

**Comment by decalage2** [comment]
I had made that change in commit aeba6ed90827829fac211b3ace2a8952f7441d60 in order to support both Python 2 and 3 (the import behaviour is different). Will have to test PR #111 

Maybe the best solution is to remove olefile from oletools, and make it a dependency to be installed by pip.

**Comment by decalage2** [comment]
Fixed by 1c84a13
